pkgs/development/haskell-modules/configuration-nix.nix

```diff
@@ -722,6 +722,7 @@ self: super: builtins.intersectAttrs super {
   cut-the-crap =
     let path = pkgs.stdenv.lib.makeBinPath [ pkgs.ffmpeg_3 ];
     in overrideCabal (addBuildTool super.cut-the-crap pkgs.makeWrapper) (_drv: {
+      libraryToolDepends = [ self.c2hs ];
```

Comment:
Is `c2hs` not in the .cabal file upstream?  If not, could you fix it by sending a PR upstream?

Also, if I remember correctly, I think the maintainer of cut-the-crap is active here in nixpkgs.  You might want to directly ping them for any reviews/questions.

Reply:
Ah I realize now I can just add it to the cabal, thanks for that, that's a much neater solution. It's fixed in [1.4.2](https://hackage.haskell.org/package/cut-the-crap-1.4.2), should build again on next update of haskell packages.